roles/nginx/templates/alfresco_proxy.j2

```diff
@@ -71,6 +70,16 @@
         proxy_set_header X-Forwarded-For   $proxy_add_x_forwarded_for;
         proxy_set_header X-Forwarded-Proto $scheme;
         proxy_pass_header Set-Cookie;
+    }
+
+    location /auth/ {
+        proxy_pass http://{{ identity_host }}:{{ ports_cfg.identity.http }}/;
+        proxy_redirect off;
+        proxy_buffering off;
+        proxy_set_header Host              $host:$server_port;
+        proxy_set_header X-Real-IP         $remote_addr;
+        proxy_set_header X-Forwarded-For   $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
         proxy_pass_header Set-Cookie;
```

Comment:
If I understood coorectly how edge mode works I think we should not pass this header at the nginx level. Those should be set by the AWS LB which endorse the SSL termintation

Reply:
nginx config should be flexible enough to do the right when there is another reverse proxy in front of it, let me do some research because the current config is a plain copy-paste of what we already have here